src/Bridge/Symfony/Bundle/Resources/views/Redoc/index.html.twig

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>{% if title %}{{ title }} - {% endif %}API Platform</title>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
```

Comment:
We try to not rely on CDNs (because they track our users, and don't work without internet). Can you get rid of this font, or install it locally?

Reply:
But Google Fonts is used here too: https://github.com/api-platform/core/blob/c90129451ca964d6a1c9681a09e540504d97ee7d/src/Bridge/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig#L7

Reply:
👍